src/deckgl-layers/layer-utils/cpu-aggregator.ts

```diff
@@ -383,19 +413,17 @@ export default class CPUAggregator {
       },
       ...opts.initialState
     };
-    this.dimensionUpdaters = {};
-    this.aggregationUpdater = {};
 
-    this._addDimension(opts.dimensions || defaultDimensions);
-    this._addAggregation(opts.aggregation || defaultAggregation);
+    this.dimensionUpdaters = opts.dimensions || defaultDimensions;
```

Comment:
remove the `_addDimension` will change the `dimensionUpdaters` to an array instead of an object